CommunityContribution.Rmd

```diff
@@ -0,0 +1,13 @@
+---
+title: "Community Contribution"
+author: Yuxin Cui
```

Comment:
The line after the title should be blank
The 3rd line is your name (without the "author:")